WebRoot/messages/ZaMsg.properties

```diff
@@ -1197,7 +1197,8 @@ NAD_zimletStatus = Enabled:
 
 NAD_CanonicalFrom = Canonical address:
 CannonicalWarning = Modifying canonical address may break mail routing and calendar appointments!
-LBL_zimbraHideInGal = Hide in GAL:
+LBL_zimbraHideInGal = Hide user in GAL:
+LBL_zimbraHideAliasesInGal = Hide aliases in GAL
```

Comment:
Just to confirm. A colon `:` is unnecessary in the value of `LBL_zimbraHideAliasesInGal`, correct?

Reply:
> Just to confirm. A colon `:` is unnecessary in the value of `LBL_zimbraHideAliasesInGal`, correct?

According to the ZCS-13220 mockup, the colon was not mentioned.

Reply:
okay